brave-core/src/test/java/com/github/kristofa/brave/DropwizardMetricsScribeCollectorMetricsHandlerExample.java

```diff
@@ -1,8 +1,9 @@
-package com.github.kristofa.brave.scribe;
+package com.github.kristofa.brave;
 
 import com.codahale.metrics.MetricRegistry;
+import com.github.kristofa.brave.SpanCollectorMetricsHandler;
 
-class DropwizardMetricsScribeCollectorMetricsHandlerExample implements ScribeCollectorMetricsHandler {
+class DropwizardMetricsScribeCollectorMetricsHandlerExample implements SpanCollectorMetricsHandler {
```

Comment:
I would say this class is no longer related to scribe, so we could rename it to `DropwizardMetricsSpanCollectorMetricsHandlerExample` and remove scribe keyword from the meter paths below. If you decide to do it, please note that this class name appears in readme files.